package.json

```diff
@@ -24,6 +24,7 @@
     "@typescript-eslint/parser": "^5.59.6",
     "eslint": "^8.41.0",
     "eslint-config-prettier": "^8.8.0",
+    "eslint-plugin-choreography-lint": "file:packages/choreography-lint",
```

Comment:
Please don't add this package at the workspace root. It may make sense to add the linter package to the `@choreography-ts/example` package, though.